ldap_users/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
# -*- coding: utf-8 -*-
# © <2010-2014> <Augustin Cisterne-Kaas>
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).

The year is still from 2010-2014? and  the author should include Elico Corp?